README.md

```diff
@@ -119,30 +115,78 @@ Marie Curie
+#### Generated live event resources
 
-#### Output
-
-- In the S3 bucket defined as input, 4 JSON files will be produced at the end:
+Once you run the lambda test event, you should see a successful response code. You will not find the event resources in the response body though. To access the event resources generated from the last step, you need to access the `CustomerImportS3Bucket` S3 bucket. Link to the S3 bucket can be found in the CloudFormation outputs.
+- In the S3 bucket, 4 JSON files will be produced at the end:
 - <EVENT_ID>-event_moderators.json
 - <EVENT_ID>-event_attendees.json
 - <EVENT_ID>-event_talents.json
 - <EVENT_ID>-event_broadcast.json
 
 These files contain all the details required to pass the login screen, and make sure your requests are authenticated.
 
-### Deploying the full stack
+Sample talent JSON file is added below for reference. Your JSON will look similar. In this sample, Michael Jackson is used as the talent for the event. Michael Jackson can use the following access url to join the event.
+```json
+{
+  "full_name":"Michael Jackson",
+  "eventId":"LiveEvent-dOtH",
+  "attendeeId":"165cre2a-a5e2-495b-88fd-d4fasd784f8a5",
+  "access_url":"https://43fsd5p0rt1j7.cloudfront.net/talent?aId=165cre2a-a5e2-495b-88fd-d4fasd784f8a5&eId=LiveEvent-dOtH&name=Michael%20Jackson",
+  "access_key":"X83cF2otcRTGHJeD5Gkr3UkX"}
+```
+
+### Start and Stop the Transcoding Lambda Broadcasting
+After the live event resources are generated, you need to start transcoding. To start or stop the transcoding task in ECS, you need to run a test event on the transcoding lambda that was created when you deployed the serverless demo. There should be a CloudFormation stack created that ends with the suffix `-broadcast` - the name of the Transcoding Lambda should be specified in this CloudFormation stack outputs.
+
+#### Using Lambda Test Event (AWS Console UI)
+1. You can **START** the broadcast task by creating a test event with the input below, then click **Test** button: 
+```
+{
+  "action": "start",
+  "meetingURL": "<your-meeting-url-to-be-broadcast>",
+  "rtmpEndpoint": "<your-rtmp-endpoint-url>"
+}
+```
+
+The broadcast URL that you provide to this start event must include all of the required parameters. The cloudformation stack output includes only the base url and passing that will cause errors in the ECS task. To find the full url please reference the `CustomerImportS3Bucket` file <EVENT_ID>-event_broadcast.json. Example broadcast url: `https://d3vj7dfsp0rt1j7.cloudfront.net/broadcast?aId=34e5fdc6-90963-4768-b389-2878i3fff3e&eId=LiveEvent-fPfdstG&accessKey=5KQIcQ5234fsdyk4CrbPl2pnVX&name=%E2%80%B9Broadcaster%E2%80%BA`. 
+
+The RTMP endpoint can be found in the cloudformation stack outputs under `MediaLivePrimaryEndpoint`. The url will look something like this `rtmp://34.187.107.987:1935/live-event-deployment-stack-livestream/primary`.
```

Comment:
If you copied this from an actual endpoint, I'd use a dummy value.

Reply:
I intentionally changed the urls to obfuscate them and that cloud formation stack is now deleted so these endpoints are no longer valid. I think it is important for the user to see an actual url. That's what I would like to see.